app/models/schema_contract/validation.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module SchemaContract
+  class Validation < ApplicationRecord
+    self.table_name = 'schema_contract_validations'
+
+    attribute :contract_name, :string
+    attribute :user_uuid, :string
+    attribute :response, :jsonb
+    attribute :status, :string
```

Comment:
I tried adding an enum for status but kept running into errors with it. Not really sure why, but I decided it wasn't worth the fight.

Reply:
What errors were you running into? I've used enums a decent bit of the time so maybe I ran into it as well

Reply:
I believe there was an error message in the logs but I don't recall what it was. The fundamental issue was that it wouldn't save status. So I'd create it 
`v = Validation.create(status: 'initialized')`
then `v.status` would be nil. IIRC `v.valid?` was still true. I assumed maybe this was because the migration needed a default value of 0 ('initialized') but that didn't help. I've also used enums in the past but not in quite a while. I'll try it again in the morning and hit you up with questions. Thanks.